src/service/referral-api/tests/FamilyHubs.ReferralApi.FunctionalTests/WhenUsingReferralsApiUnitTests.cs

```diff
@@ -489,6 +489,36 @@ public async Task ThenReferralsByCompositeKeysAreRetrieved()
         retVal.Items.Count.Should().BeGreaterThan(0);
     }
 
+    [Theory]
+    [InlineData(1, 1)]
+    [InlineData(2, 0)]
+    public async Task Then_ReferralCount_ByServiceId_IsRetrieved(int serviceId, int expected)
+    {
+        if (!IsRunningLocally() || Client == null)
+        {
+            // Skip the test if not running locally
+            Assert.True(true, "Test skipped because it is not running locally.");
+            return;
+        }
```

Comment:
I wonder if we can add a ticket to the tech debt Lucid board around fixing this. Seems a bit mad that we have this in every test!

Reply:
100% !!

Reply:
I thought we already did fix this? I removed it in at least one other project.
Do we know if this would fail in the pipeline? If not: remove this.

Reply:
It does fail in the pipeline sadly, I tested it without before adding it back in :(